ravenframework/Models/PostProcessors/Validations/Representativity.py

```diff
@@ -0,0 +1,489 @@
+    return datasets
+
+  def _computeErrors(self,datasets,features,targets):
+    """
+      A utility function to transform variables to the relative error of these variable
+      @ In, datasets, xarray datasets, datasets containing prototype (mock) data and target data
+      @ In, features, names of feature variables: measurables
+      @ In, targets, names of target variables: figures of merit (FOMs)
+      @ out, datasets, xarray datasets, datasets after computing errors in each variable
+    """
+    for var in [x.split("|")[-1] for x in features + targets]:
+      datasets['err_'+str(var)] = (datasets[var].values - datasets[var].attrs['meanValue'])/datasets[var].attrs['meanValue']
+
+  def _computeUncertaintyMatrixInErrors(self, data, parameters):
+    """
+      A utility function to variance and covariance of variables in the error space
+      @ In, data, xarray dataset, data containing either prototype (mock) data or target data
+      @ In, parameters, names of parameters/inputs to each model
+      @ out, uncertMatr, np.array, The variance covariance matrix of errors
+    """
+    uncertMatr = np.zeros((len(parameters), len(parameters)))
+    for i, var1 in enumerate(parameters):
+      for j, var2 in enumerate(parameters):
+        if var1 == var2:
+          uncertMatr[i, j] = data[var1].attrs['var']
+        else:
+          uncertMatr[i, j] = data[var1].attrs['cov_'+var2][0,1]
+    return uncertMatr
+
+  def _calculateBiasFactor(self, normalizedSenExp, normalizedSenTar, UparVar, UmesVar=None):
+    """
+      A utility function to compute the bias factor (i.e., representativity factor)
+      @ In, normalizedSenExp, np.array, the normalized sensitivities of the mock/prototype measurables
+      @ In, normalizedSenTar, np.array, the normalized sensitivities of the target variables/Figures of merit (FOMs) with respect to the parameters
+      @ In, UparVar, np.array, variance covariance matrix of the parameters error
+      @ In, UmesVar, np.array, variance covariance matrix of the measurables error, default is None
+      @ Out, r, np.array, the representativity (bias factor) matrix neglecting uncertainties in measurables
+      @ Out, rExact, np.array, the representativity (bias factor) matrix considering uncertainties in measurables
+    """
+    # Compute representativity (#eq 79)
+    r = (sp.linalg.pinv(sqrtm(normalizedSenTar @ UparVar @ normalizedSenTar.T)) @ sqrtm(normalizedSenTar @ UparVar @ normalizedSenExp.T) @ sqrtm(normalizedSenTar @ UparVar @ normalizedSenExp.T) @ sp.linalg.pinv(sqrtm(normalizedSenExp @ UparVar @ normalizedSenExp.T))).real
+    rExact = (sp.linalg.pinv(sqrtm(normalizedSenTar @ UparVar @ normalizedSenTar.T)) @ sqrtm(normalizedSenTar @ UparVar @ normalizedSenExp.T) @ sqrtm(normalizedSenTar @ UparVar @ normalizedSenExp.T) @ sp.linalg.pinv(sqrtm(normalizedSenExp @ UparVar @ normalizedSenExp.T + UmesVar))).real
```

Comment:
The default for UmesVar is None, which will error out if using the default value.